atomic-pkglayer.spec

```diff
@@ -0,0 +1,38 @@
+Summary: A tool to install debug packages on Atomic Host
+Name: atomic-pkglayer
+Version: 2015.3
```

Comment:
Looks like I fucked up the tag =(  It's not pointing to a commit reachable from master.  Sigh...will fix it with v2016.1.